src/js/components/BuildSection.jsx

```diff
@@ -210,7 +214,19 @@ const BuildSection = ( {setLogs, hardwareType, setHardwareType} ) => {
         }
 
         if ( newValue.value.parameters ) {
-            setCommandParameters(JSON.stringify(newValue.value.parameters, null, 4));
+            // prepare parameters for editing
+            setCommandParameters(
+                JSON.stringify(
+                    convertCommandParametersFromCodecFormat({
+                        id: command.value.id,
+                        type: commandType,
+                        direction: command.value.directionType,
+                        parameters: newValue.value.parameters
+                    }),
+                    null,
```

Comment:
add universal replacer which detect any special classes like Uint8Array and remove convertCommandParametersFromCodecFormat

Reply:
done